### DISCUSSION TITLE
How to disable the extra line at the end of a text?

### DISCUSSION THREAD

**Discussion by serg-vinnie** [krzyzanowskim/STTextView#47] [Questions]
Hi,

STTextView was created as a text editing tool and adding the extra line at the end of a text is completely reasonable step.
But it is completely unnecessary for a static text to display. In my opinion the extra line should be disabled alongside with the editing capability. So 'isEditable = false' should also disable the extra line as well.

But regardless of our agreement/disagreement of the isEditable behaviour I would like to know the exact place in the code which is responsible for the extra line.

Cheers!

**Comment by krzyzanowskim** [comment]
there is no extra line added. at least I don't see such. how to reproduce?

![Screenshot 2024-05-10 at 22 47 55](https://github.com/krzyzanowskim/STTextView/assets/758033/a0f2c404-df89-4ff0-ac78-0e529fa11923)

Note: there is an extra line concept in the text layout, but then again it has no representation in the storage.

**Comment by serg-vinnie** [comment]
the last row is not numbered but it can be selected with mouse click. Can you confirm your last row is selectable? 
Otherwise I will prepare some example code

**Reply by krzyzanowskim** [reply]
I can't reproduce. Can you record video maybe? Maybe it is already fixed on the main branch
